build.cake

```diff
@@ -198,9 +162,9 @@ Task("BuildIntegrationTests")
 Task("PackageDockerIntegrationTests")
   .Does(() =>
  {
-    Zip("./tools/XComponent.Build.Community/tools/XCBuild/XCRuntime", "./docker/integration_tests/dockerScripts/XCContainer/XCRuntime.zip");
+    Zip( GetXcRuntimePath().Replace("xcruntime.exe", ""), "./docker/integration_tests/dockerScripts/XCContainer/XCRuntime.zip");
```

Comment:
As you need it, I will add a method in Cake.XComponent to get the directory of the Runtime

Reply:
yes, i will remove the code when you do it

Reply:
So it will be done in another PR